choc-pizza.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Chocolate Pizza
+		</title>
+		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
+		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand' rel='stylesheet' type='text/css' />
+	</head>
+	<body class="container_16 clearfix">
+		<header id="top-header" class="grid_15">
+			<div id="top-logo" class="toplogo">
+				<img src="img/logo.png" />
+			</div>
+			<div id="title-plus-subtitle" class="toplogo">
+				<div id="delicious">Delicious
```

Comment:
Dexter,

I am working on these assignments and implementing your suggestions at this
very moment. Intend to make resubmissions. Thank you.

I think the reason I used so many divs was, I realized I needed some extra
tags once I was doing the css, and I intended to rename all the div's later
and forgot. I am working on that right now.

On Mon, Jan 19, 2015 at 2:24 PM, Dexter Lesaca <notifications@github.com>
wrote:

> In choc-pizza.html
> <https://github.com/jayspringate/chocolate-pizza/pull/2#discussion-diff-23191012>
> :
>
> > +	<head>
> > +		<title>
> > +			Chocolate Pizza
> > +		</title>
> > +		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
> > +		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
> > +		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
> > +		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>' rel='stylesheet' type='text/css' />
> > +	</head>
> > +	<body class="container_16 clearfix">
> > +		<header id="top-header" class="grid_15">
> > +			<div id="top-logo" class="toplogo">
> > +				<img src="img/logo.png" />
> > +			</div>
> > +			<div id="title-plus-subtitle" class="toplogo">
> > +				<div id="delicious">Delicious
>
> A div has little semantic meaning. Please review the use of heading tags
> such as h1 etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191012>.
>